packages/docs/concepts/connections-and-requests.yaml

```diff
@@ -177,6 +177,273 @@ _ref:
                             _request: request1
             ```
 
+            ## Working Example
+
+            For this example, a connection to [{JSON}Placeholder](https://jsonplaceholder.typicode.com/) will be used.
+            To create this `connection`, define the connection as follows in the `lowdefy.yaml` file:
+
+            ###### lowdefy.yaml
+            ```yaml
+            lowdefy: LOWDEFY_VERSION
+
+            connections:
+              - id: my_api
+                type: AxiosHttp
+                properties:
+                  baseURL: https://jsonplaceholder.typicode.com
+
+            pages:
+              - id: home
+                type: PageHeaderMenu
+            ```
+
+            To use this connection, a `request` must be defined, which references the corresponding `connectionId`. A request to get the list of users can be added to the home page as follows:
+
+            ###### lowdefy.yaml
+            ```yaml
+            lowdefy: LOWDEFY_VERSION
+
+            connections:
+              - id: my_api
+                type: AxiosHttp
+                properties:
+                  baseURL: https://jsonplaceholder.typicode.com
+
+            pages:
+              - id: home
+                type: PageHeaderMenu
+
+                requests:
+                  - id: get_users
+                    type: AxiosHttp
+                    connectionId: my_api
+                    properties:
+                      url: /users
+            ```
+
+            This request should be called when the page is mounted. To do this, the `Request` action will be used during the `onMount` event of the home page.
```

Comment:
Change to [`Request`](/Request) .. and `onMount` event of the home page[`PageHeaderMenu`](/PageHeaderMenu) block.

Reply:
Change made in d611ea5901b9c9e552b83c41f6a7f5b674515f34